src/extension.ts

```diff
@@ -421,14 +437,14 @@ function maybeRunKubernetesCommandForActiveWindow(command) {
                         vscode.window.showErrorMessage("Save failed.");
                         return;
                     }
-                    kubectl.invoke(`${command} "${editor.document.fileName}"`);
+                    runKubectlCommandInTerminal("kubectl", `${command} "${editor.document.fileName}"`);
```

Comment:
I feel like we're losing some of the encapsulation of the `kubectl` module here.  Have you considered a design where you provide e.g. an `invokeOptions` argument to `kubectl.invoke`, and if the caller specifies a terminal then run it in the terminal?  Open to other ideas too!

Reply:
I prefer to add a new method `invokeInTerminal` in `kubectl` interface.